packages/agents-manager/src/hooks/use-chat-layout-manager/style.scss

```diff
@@ -517,4 +534,26 @@ body.agents-manager-sidebar-container {
 		background-color: $gray-700;
 		color: $gray-400;
 	}
+
+	.agents-manager-selected-block {
+		color: $white;
+		border: 1px solid rgba( $white, 0.1 );
+		margin-top: 8px;
+
+		&__divider {
+			border-left: 1px solid rgba( $white, 0.1 );
+		}
+
+		&__remove {
+			&.components-button {
+				color: $white;
+			}
+		}
+	}
+}
+
+/* Styles for the undocked chat */
+.agents-manager-chat--undocked .agenttic {
+	// Fix the undocked chat overlapping with the nav menu
+	z-index: 9999;
```

Comment:
This isn't working in Calypso.
<img width="2245" height="2455" alt="CleanShot 2025-11-24 at 6  00 04" src="https://github.com/user-attachments/assets/745cdce9-ec8f-4ec6-895e-11962d267060" />

Issue: https://linear.app/a8c/issue/DOTCOM-15300/floating-chat-behind-calypso-menu